docs/root/start/quick-start/_include/envoy-demo-tls-sni.yaml

```diff
@@ -60,3 +60,11 @@ static_resources:
       typed_config:
         "@type": type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext
         sni: www.envoyproxy.io
+        common_tls_context:
```

Comment:
I added this because I didn't want an example to not have a `validation_context`. That makes it a dangerously-insecure example.